server/src/parsing/error-parsing.ts

```diff
@@ -242,13 +242,13 @@ export function parseError(txt: AnnotatedText): AnnotatedText {
         x: string;
         y: string;
         mid: [number, number];
-        diff: diff.IDiffResult[];
+        diff: diff.Change[];
         cost: number;
       }[] = [];
       for (let mid of indicesOf(str, df.mid, preLen, str.length - postLen)) {
         const x = str.substring(preLen, mid[0]);
         const y = str.substring(mid[1], str.length - postLen);
-        const xy = diff.diffWords(x, y);
+        const xy = diff.diffLines(x, y);
```

Comment:
What about this change? Was it done on purpose?

Reply:
Yes. IIRC, there is no `diffWords` in the new API.

Reply:
Edit: I'm feeling unsure about this. Maybe it's better to use the new `diffWords` API to minimize the changes? See [ref](https://github.com/kpdecker/jsdiff).